warehouse/models/intermediate/gtfs/gtfs_quality/_int_gtfs_quality.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+
+models:
+  - name: int_gtfs_quality__schedule_feed_guideline_index
+    description: |
+      Daily index of v2 schedule feeds. Provides structure for
+      individual check models to be joined together by day.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - feed_key
+            - date
+
+  - name: int_gtfs_quality__schedule_validation_severities
```

Comment:
Should there be an equal row count test between each implemented check and the feed guideline index? Part of the point of the old model was to ensure that we weren't getting fanout OR missing rows, which is why there was an equal row count between the index and the final fact table AND there was the absolute not_null.

Reply:
Yes good call, will add rowcounts.